layers/+lang/python/funcs.el

```diff
@@ -91,6 +91,11 @@ when this mode is enabled since the minibuffer is cleared all the time."
   (setq mode-name "Python"
         tab-width python-tab-width
         fill-column python-fill-column)
+
+  ;; since we changed the tab-width we need to manually cally python-indent-guess-inden-offset herr
+  (when python-spacemacs-indent-guess
+    (python-indent-guess-indent-offset))
```

Comment:
Does this run whenever an user opens a new Python buffer? I guess not.

Reply:
it does run whenever you open a buffer (it's part of `python-mode-hook`)